packages/core/test/VDocument.test.ts

```diff
@@ -615,6 +615,177 @@ describe('stores', () => {
                         });
                     });
                 });
+                describe('Lists', () => {
```

Comment:
I'm so looking forward to when these tests will be moved to their respective plugins rather than cluttering VDocument... :)